types/jsrsasign/jsrsasign-tests.ts

```diff
@@ -153,3 +153,60 @@ x509crl.getRevCertArray(); // $ExpectType RevokedCertificate[] | null
+            "sighex": "39d5989b2ae6b62357c450997906ed63454ac90db549c88fa6e8033ea25bae05cfacf275de1225cac8ce0fa1c1107f62252768c0327b9ce45c6c7a4b2354ac2a3fb1a0e90454e817aed0289bc70e81cd8211af85d76eb739d3fe9460ccdf6d40f4f05e4003f84e9263318bf118b16731fa04964b88630e9d4d7cd1e0c22afd6fd52229e72022dd834d783135c9c5da66bf5359e7f7c1b61fa1b353957969a0e134542bbdd61a59e3c46bb5902a83e265bc7b5a5c2dba03c32066978af19951845cb8aa97c90961a735156b64447ef890ed6db22f1b2db098dd93b9d149a6672cc78d4349bb9f0d6f1340c96f1bcfbc24de5fba8f970364e8e973eb78dd33717b"
+        }
+    ],
+    "certs": [
+        "-----BEGIN CERTIFICATE-----\r\nMIIDmjCCAoKgAwIBAgIUPm2DqtwN0/JOREckQbYhHI60iP8wDQYJKoZIhvcNAQEL\r\nBQAwGTEXMBUGA1UEAwwOZmFrZWNhLml1ZHguaW8wHhcNMjQxMDI5MDkxNjM0WhcN\r\nMjUxMDI5MDkxNjM0WjAvMQswCQYDVQQGEwJJTjEgMB4GA1UEAwwXdGVzdC5haXBA\r\nZGF0YWthdmVyaS5vcmcwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCz\r\nFgh08JTzYYiSGBHtKQUsinOsjRMGsGEeP7m4TFTODEsyUfItuQrWsi9LRtzrFigq\r\nOde+bI67aBUj3krM2JfPe/1aDASnS8tjff9A9IrX5y9rAdaa0rZBPtWaNWai+s1/\r\ncD5CIb4rB5PHcJL/SnlBgbKtBylP4d/mRO+m/abx3tMXuOvEiGufhrCDrz6NT/B4\r\npqln+ChdpNb/1N4GLHYhb0aRXa+eHqWliJW5+TJB6Y3aNUutO2Yc02Sa5S4GD6xD\r\ny7nzTxWzrS93V+7fvFZk+DL3xZyY9gXD3YfW9EjG53gwbS3WiC4PBeJqUfR3HJnI\r\ngfIfsuNmZumufMjeECDvAgMBAAGjgcMwgcAwDgYDVR0PAQH/BAQDAgbAMB0GA1Ud\r\nDgQWBBTcnrsUPuS9CSQrV0YIKI+4P97atTAfBgNVHSMEGDAWgBQQmPVY2VIxLDbC\r\nr491i0Xlz9ECZzAlBgNVHREBAf8EGzAZgRd0ZXN0LmFpcEBkYXRha2F2ZXJpLm9y\r\nZzBHBgNVHR8BAf8EPTA7MDmgJaAjhiFodHRwczovL2Zha2VjYS5pdWR4LmlvL2Zh\r\na2VjYS5jcmyiEIIOZmFrZWNhLml1ZHguaW8wDQYJKoZIhvcNAQELBQADggEBAFTs\r\n/W/ONQebv+tN0A52AtE3YZ4WXcwHHi0wF4MFQV/qmEygmV8ZPJQ+6x5z02KnDLcE\r\nXSFWH5b0Fhrj7OPB5Smnnd47syRVuDHIteAFdx8c2Ds65Mrui2Rv1yZnepwTwVtz\r\n1S6pRV/G4C8Ym91s27pfhFa9Pj5LvGgviZiFOJ/GibJcm9n19UIn1qFMg6TYTmkl\r\n6NzAZK2r2icNtWuSHWFY2rD2LEH3dorduUrfX+ykxUwVPDb7DQs4JJHu8h3+8AlM\r\nkSegyo4FGaXAmTHvMUW7d626WGyZIBWVnWFqyqY4p3BvoAl3ByxCTtUiKCb/jNGm\r\nq2EAY77Z7TqDLdfzVlY=\r\n-----END CERTIFICATE-----\r\n"
```

Comment:
The actual content of these certificates is unnecessary for the purpose of the test as no code is actually executed. The value `""` would be sufficient this test.